includes/form.html

```diff
@@ -13,6 +13,7 @@
 {%- assign formUncentered = formForm.uncentered | default: include.uncentered | default: ''-%}
 {%- assign formReplaceValue = include.replaceValue -%}
 {%- assign formOptions = "" | split: "" -%}
+{%- assign formUuid = include.uuid | default: false -%}
```

Comment:
Das mit den "defaults" kannst du dir sparen. Hatte ja mal gesagt, keine zusätzliche Logik mehr in Liquid. Wenn ich die Vue Form benutze, hat sie die Logik auch nicht. Also so einfach wie möglich.